apps/codeforafrica/src/components/RichText/RichText.js

```diff
@@ -97,7 +97,7 @@ const RichText = React.forwardRef(function RichText(props, ref) {
   }
   return (
     <Box color={color} sx={sx} ref={ref}>
-      {serialize(elements, other)}
+      {serialize(elements, { ...other, sx: { typography: sx?.typography } })}
```

Comment:
I kind get what you're trying to do here but I think it's too "hack-y". Let's just introduce a prop for it e.g. `typographyProps` so that someone can use it as
```jsx
   <RichText elements={...} typographyProps={{ sx: {...}, ... }}  sx={...} /> 
```

`sx` will be applied to the root `Box` but `typographyProps` will be based to `serialize` as is.